app/Models/Role.php

```diff
@@ -170,7 +243,39 @@ public static function possibleObjectsFor($name)
             return self::toSelectableCollection($collegists);
         }
 
-        return [];
+        return collect([]);
+    }
+
+    public function hasElevatedPermissions(): bool
+    {
+        return in_array($this->name, [self::PRINT_ADMIN, self::NETWORK_ADMIN, self::PERMISSION_HANDLER, self::SECRETARY, self::DIRECTOR]);
+    }
+
+    public function hasTranslatedName(): bool
+    {
+        return in_array($this->name, [self::WORKSHOP_ADMINISTRATOR, self::WORKSHOP_LEADER]);
+    }
+
+    /**
+     * Create a collection with id and name of the items in an array.
+     * The ids starts at 1.
+     * @param array $items the items in the array will be the name attributes
+     * @return collection
+     */
+    private static function toSelectableCollection(array $items)
```

Comment:
I prefer to keep private functions at the bottom of the page. (only my preference)